ignite/pkg/cosmosgen/templates/root/client.ts.tpl

```diff
@@ -36,6 +43,7 @@ export class IgniteClient {
 		return await signingClient.signAndBroadcast(address, msgs, fee ? fee : defaultFee, memo)
   }
   constructor(env: Env, signer: OfflineSigner) {
```

Comment:
@clockworkgr Shouldn't we mark the `signer` parameter as optional now we have this `useKeplr` feature?